src/vaxis.zig

```diff
@@ -2,23 +2,36 @@ const std = @import("std");
 const builtin = @import("builtin");
 const atomic = std.atomic;
 const base64 = std.base64.standard.Encoder;
+const posix = std.posix;
+
+pub const Cell = @import("Cell.zig");
+pub const Image = @import("Image.zig");
+pub const InternalScreen = @import("InternalScreen.zig");
+pub const Key = @import("Key.zig");
+pub const Mouse = @import("Mouse.zig");
+pub const Options = @import("Options.zig");
+pub const Queue = @import("queue.zig").Queue;
+pub const Screen = @import("Screen.zig");
+pub const Tty = @import("Tty.zig");
+pub const Window = @import("Window.zig");
+pub const Winsize = Tty.Winsize;
+pub const Style = Cell.Style;
+
+pub const ctlseqs = @import("ctlseqs.zig");
+pub const gwidth = @import("gwidth.zig");
+pub const widgets = @import("widgets.zig");
 
-const Queue = @import("queue.zig").Queue;
-const ctlseqs = @import("ctlseqs.zig");
-const Tty = @import("Tty.zig");
-const Winsize = Tty.Winsize;
-const Key = @import("Key.zig");
-const Screen = @import("Screen.zig");
-const InternalScreen = @import("InternalScreen.zig");
-const Window = @import("Window.zig");
-const Options = @import("Options.zig");
-const Style = @import("Cell.zig").Style;
-const Hyperlink = @import("Cell.zig").Hyperlink;
-const gwidth = @import("gwidth.zig");
-const Shape = @import("Mouse.zig").Shape;
-const Image = @import("Image.zig");
```

Comment:
I think I would prefer to keep these all private at this level and only expose these via `main.zig`. If there are any missing that are needed, let's add them there (but only if actually needed so as to keep the API small)